### PR TITLE
Minha parte da história

### PR DESCRIPTION
**Pull request by emozart**
Please review this, @casey-collab.
Thank you.

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2207)

### CODE REVIEW

File: portugues/colabora/comdiferencas/diff/pull/reenvio/watch/divulga/colaborador/colaborador.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Você continua atuando apenas como colaborador no projeto. Por causa da sua boa contribuição
```
Comment (casey-collab) [inline portugues/colabora/comdiferencas/diff/pull/reenvio/watch/divulga/colaborador/colaborador.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: portugues/colabora/comdiferencas/diff/pull/reenvio/watch/divulga/instrutor/instrutor.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Você desenvolve um curso de Git e GitHub a passa a ganhar uma boa grana com
```
Comment (casey-collab) [inline portugues/colabora/comdiferencas/diff/pull/reenvio/watch/divulga/instrutor/instrutor.md]: This file has also been added to a directory that already contained a page.

**Comment by casey-collab** [comment]
It looks like the files `portugues/colabora/comdiferencas/diff/pull/reenvio/watch/divulga/colaborador/colaborador.md~` and `portugues/colabora/comdiferencas/diff/pull/reenvio/watch/divulga/instrutor/instrutor.md~` are empty.  Please make sure you don't add empty files to the repository.  You could add one sentence to each file, or delete the options that lead to them.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by emozart** [comment]
Hi, @casey-collab.

Take a look again, plase.

Thank you.

**Comment by casey-collab** [comment]
Looks great! :dog:

**Pull request merged**